apps/wallet-mobile/src/components/Modal/ModalScreen.tsx

```diff
@@ -55,7 +61,9 @@ export const ModalScreen = () => {
           </View>
         </Animated.View>
       </KeyboardAvoidingView>
-    </View>
+
+      <View style={[styles.fixBottomColor, {height: bottom}]} />
```

Comment:
fix the space between the modal + safe are view, otherwise the modal looks like a sandwich between the backdrop.